src/components/Column.js

```diff
@@ -43,60 +43,29 @@ ColumnContainer.defaultProps = {
   divisions: 12,
 };
 
+const compute = name => props =>
+  ((divisions, size, shift) => `
+    ${size ? `width: ${divvy(divisions, size)}%;` : ''}
+    ${shift ? `margin-left: ${divvy(divisions, shift)}%;` : ''}
+  `)(props.divisions, props[name], props[`${name}Shift`]);
+
+const breakpoint = name =>
```

Comment:
Since there's two identical `breakpoint` functions, how about we externalize this into `utils/breakpoint`? Also, since the `compute` functions are different, maybe change the args to accept an `innerStyle` string; that way we could pass `breakpoint(compute('xs'))`.

Let me know what you think about that 👍